### PR TITLE
JoeBacus-Intro-Python-I

### PR DESCRIPTION
**Pull request by joepounddev**
All MVP and stretch tasks completed. Linting (Pylint, Flake8 & PEP8) and PEP8-compliant formatting (Autopep8) configured from the start of the project.

### CODE REVIEW

File: src/05_lists.py [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+del x[4]
```
Comment (kjohna) [inline src/05_lists.py:21]: .remove() is an option here as well (but del does have better performance if you know the index ahead of time)

Comment (joepounddev) [reply src/05_lists.py:21]: Thanks! That's handy, will keep that in mind. :)

File: src/08_comprehensions.py [hunk 2]
```diff
@@ -36,6 +36,6 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
+y = [int(n) for n in x if not int(n) & 1]
```
Comment (kjohna) [inline src/08_comprehensions.py:39]: oh that's fun. I feel like I only see (and do) `%2 == 0`

File: src/09_dictionaries.py [hunk 3]
```diff
@@ -35,10 +35,23 @@
...
+        waypoint["name"] = "a real place"
+        break
 
 # Write a loop that prints out all the field values for all the waypoints
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+for waypoint in waypoints:
+    for key in waypoint:
+        print("%-5s-> %s" % (key, waypoint[key]))
```
Comment (kjohna) [inline src/09_dictionaries.py]: ooo that makes it look real nice.

Comment (joepounddev) [reply src/09_dictionaries.py]: Thank you! :)

File: src/08_comprehensions.py (hunk 2 above)
Comment (joepounddev) [reply src/08_comprehensions.py:39]: Yeah bitwise is fun, though I think I'll have to be cautious if I ever encounter a new data type. It seems to work well enough with integers ~and decimals~ in my experience, at least.

**Pull request merged**